spec/interactors/editions/create_interactor_spec.rb

```diff
@@ -69,7 +69,7 @@
     end
 
     context "when the edition is live" do
-      let(:edition) { create(:edition, live: true, number: 2) }
+      let(:edition) { create(:edition, live: true, number: 2, published_at: Time.zone.now) }
```

Comment:
Could you just do `create(:edition, :published, number: 2)` here? As the published trait sets live to true and populates the relevant timestamps like `published_at`

Reply:
I'd wonder if this test and the one above about discarded now need adapting as they're logic is no longer covered by this test and should be covered by the service unit tests (I'm thinking we may have missed a test that the number is changed).

We may just need to assert here that the service is called.

Reply:
@1pretz1 Thanks - good spot, have updated accordingly for now.  But may change due to @kevindew 's comment.  Will leave this outstanding for now.

Reply:
@kevindew Have re-jigged things - hopefully in the right way.